operator/builtin/output/elastic/elastic.go

```diff
@@ -125,7 +126,42 @@ func (e *ElasticOutput) Stop() error {
 	e.cancel()
 	e.wg.Wait()
 	e.flusher.Stop()
-	return e.buffer.Close()
+
+	entries, err := e.buffer.Close()
+	if err != nil {
+		e.Error("Failed to retreive entries")
+		return err
+	}
+
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*5)
+	defer cancel()
+
+	err = e.sendEntries(ctx, entries)
```

Comment:
Looks like we aren't protecting against len(entries) == 0, so it looks like we'll send a request with 0 entries?